src/Models/KeyResult.php

```
@@ -104,6 +104,15 @@ public static function findByProjectOkrs(Database $db, int $projectId, int $orgI
               ORDER BY hwi.priority_number ASC, kr.display_order ASC", [':pid' => $projectId, ':oid' => $orgId])->fetchAll();
     }
 
+    public static function workItemBelongsToOrg(Database $db, int $workItemId, int $orgId): bool
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)